### ISSUE TITLE
Inline selection is broken (slate@0.47)

### ISSUE THREAD

**Issue by vashchukmaksim** [ianstormtaylor/slate#2779]
#### Do you want to request a _feature_ or report a _bug_?

Bug

#### What's the current behavior?

Run **examples/links** and try to select "hyperlinks" link in the editor area. It throws an error:

```js
element.js:2583 Uncaught Error: `Node.assertNode` could not find node with path or key: List [ 0, 1, 0 ]
    at Block.ElementInterface.<computed> [as assertNode] (element.js:2583)
    at Block.createIterable (element.js:165)
    at Block.descendants (element.js:365)
    at Block.texts (element.js:1875)
    at findSelection (queries.js:486)
    at onQuery (queries.js:36)
    at next (editor.js:304)
    at onQuery (queries.js:35)
    at next (editor.js:304)
    at onQuery (queries.js:35)
```

![Screen Recording 2019-05-15 at 12 43 07](https://user-images.githubusercontent.com/26834615/57766153-86207680-770f-11e9-9e01-63f13710a9f4.gif)

I catch this error at the same situation in my own app, then checked online demo and then cloned slate monorepo and ran example locally. The error is presented everywhere.

#### What's the expected behavior?

No errors

**Comment by adjourn** [comment]
Im pretty sure it was fixed by #2764 (not released yet). [Test it here](https://deploy-preview-2764--slatejs.netlify.com/#/links).

**Comment by vashchukmaksim** [comment]
@adjourn Unfortunately it is still there

**Comment by adjourn** [comment]
Ah yes, managed to replicate it with double click. Doesn't seem to be related to #2764.

**Comment by vashchukmaksim** [comment]
Yeah, now only double-click throws this error but part selection seems to be fixed with #2764

**Comment by audunhalland** [comment]
Duplicate of #2769

**Comment by adjourn** [comment]
Is it fixed and can be closed now? #2799 "fixes" was not correctly used.

**Comment by vashchukmaksim** [comment]
Yes, it seems that it's working properly with those fixes. I'm closing this issue then.
